bin/import_xml2obd.py

```diff
@@ -340,20 +397,35 @@ def pull_docs(conf_file, local_dir):
     src = RemoteStorage(src_user, src_key, src_container)
 
     all_docs = src.new_docs()
+    print 'total source files ', len(all_docs)
     xmls = [x for x in all_docs if x[-4:]=='.xml' and x[:-4] in all_docs and
             x[:-4] + '.ind' in all_docs]
     docs = [x[:-4] for x in xmls]
     inds = [x + '.ind' for x in docs]
     files = xmls + docs
     for fname in files:
         print 'Downloading ',fname
-        src.download_blob(fname, local_dir + '/' + fname.split('/')[-1])
+        obj = src.get_obj(fname)
+        localname = local_dir + '/' + fname.split('/')[-1]
+        objmd5 = base64md5str(obj.properties.content_settings.content_md5)
+        if os.path.isfile(localname) and objmd5 == md5str(localname):
```

Comment:
It might be more readable. I could save (filename, md5) pairs in a separate local file under the same local directory.